zap/src/output/luau/client.rs

```diff
@@ -89,6 +89,7 @@ impl<'src> ClientOutput<'src> {
 			self.config.event_id_ty().size()
 		));
 	}
+
```

Comment:
I don't think this diff is worth it tbh for this file. I'm happy to keep the formatting in `base.luau` though